bin/test_rhel8_integration

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash -ex
+
+cleanup() {
+  echo "Cleaning up..."
+  echo "Cleaning up... logs in cleanup.log"
+  docker-compose rm --stop --force
+}
+
+DEBUG="false"
+if [[ "$1" = "--debug" || "$1" = "-d" ]]; then
+  DEBUG="true"
+  shift
+fi
+
+trap 'echo "ERROR: Test script encountered an error!"; docker-compose logs &> cleanup.log; cleanup' ERR
+trap 'cleanup' EXIT
+cleanup
+
+echo "Building API container..."
+docker-compose build
+
+echo "Generating certificate..."
+docker-compose up openssl
+
+echo "Starting Conjur..."
+docker-compose up -d conjur-appliance
+
+
+echo "Configuring Conjur..."
+export ADMIN_PASSWORD=P@S5-ITAbXJlxH1MU
+export PYTHON_KEYRING_BACKEND=keyring.backends.SecretService.Keyring
+export DEBUG=$DEBUG
+export CONJUR_AUTHN_API_KEY=P@S5-ITAbXJlxH1MU
+export SERVER_MODE="appliance"
+export ACCOUNT="dev"
+export LOGIN="admin"
+export TEST_HOSTNAME=conjur-appliance
```

Comment:
I worry about this because we have this in constants.py. I agree that this value should be here instead of in the constants.py so in the test_integration file can you add `export TEST_HOSTNAME=conjur-https`